### ISSUE TITLE
Built version does not match source code

### ISSUE THREAD

**Issue by alistairjcbrown** [addthis/fluxthis#99]
The built file on npm appears to differ from the source code for v2.3.2

__Built file contains:__
```js
if (response.type.indexOf('application/json') === 0) {
```

__Source code contains:__ (in `implore.es6.js`)
```js
if (response.type === 'application/json') {
```

This causes an issue when upgrading as we manage the JSON parsing in the success callback -`type` contains a charset value, so the body wasn't automatically parsed by `fluxthis`.

**Comment by alistairjcbrown** [comment]
Looks like this affects all 2.3.x versions - I'll use 2.2.6 in the meantime.

**Comment by engenb** [comment]
I'm experiencing this issue as well.  Please review #101

**Comment by alistairjcbrown** [comment]
@engenb I think the easiest solution here is to push a patch version bump with a new build which reflects the code current in the code base. Changing how the Content-Type header is checked can come after.
@jhorwit2 is is possible to push a new build, one which matches the current source code, and which uses the expected `response.type === 'application/json'` ?

**Comment by jhorwit2** [comment]
Let me review the failing test on master & i'll take care of publishing a new version with the PR. Sorry about the delay.

**Comment by jhorwit2** [comment]
@alistairjcbrown @engenb this should be fixed with v2.4.0 & with the application charset support.

**Comment by engenb** [comment]
awesome - thank you!
